indexer/README.md

```diff
@@ -2,225 +2,119 @@
-
-Your `<callback_url>` should be able to take post requests from the Blue Box and index the resulting files 
-This is untested, but can take in a simulated curl request of the following format.
-```
-curl -H "Content-Type: application/json" -X POST -d '{ "query": { "query": { "bool": { "must": [ { "match": { "files.sample_json.donor.species": "Homo sapiens" } }, { "match": { "files.assay_json.single_cell.method": "Fluidigm C1" } }, { "match": { "files.sample_json.ncbi_biosample": "SAMN04303778" } } ] } } }, "subscription_id": "ba50df7b-5a97-4e87-b9ce-c0935a817f0b", "transaction_id": "ff6b7fa3-dc79-4a79-a313-296801de76b9", "match": { "bundle_version": "2017-08-03T041453.170646Z", "bundle_uuid": "4ce8a36d-51d6-4a3c-bae7-41a63699f877" } }' <callback_url> 
-```
-
-### Methods and Endpoints
+    ```
+9. Then, replace the following values in the config.json with your own values. 
+    - `<your-indexer-lambda-application-name>` is the name that you will give the lambda project.
+    - `<AWS-account-ID>` is one of the values in the access policy of your Elasticsearch. An numerical id of your AWS Account. 
+    - `<your blue box>` is the endpoint of the DSS Blue Box.
+    - `<your elasticsearch endpoint>` is url you will use to access Azul's Elasticsearch Domain. This url can be found by returning to your Elasticsearch Service Dashboard,
+    clicking the link of your Elasticsearch Domain. The url should be labled as **Endpoint** in the Overview Tab.
+10. Next, like the config.json. Open `~/.profile`. If that file doesn't exist, create one and append the text below. 
+    ```
+    export ES_ENDPOINT=<your elasticsearch endpoint>
+    export BLUE_BOX_ENDPOINT=<your blue box>
+    export ES_INDEX=<elasticsearch index to use>
+    export INDEXER_NAME=<your-indexer-lambda-application-name>
+    export HOME=/tmp
+    ```
+11. Then, replace the `<>` values in the `.profile`. They are the same values used in the `config.json` from the previous steps.
+11. Run `. ~/.profile` to load the variables
+
+### 4. Modifying IAM Access Policies
+
+1. In the AWS Console, click Services in the top toolbar, and click IAM in the Security, Identity & Compliance subsection.
+2. On the side menu bar, click **Roles**, then search for your lambda function, `<your-indexer-lambda-application-name>`. Once you find it, click on it.
+3. Find a table in the Permissions tab under the blue Attack Policy button. You should see for lambda name in the table. Click on the caret on the right side of the lambda name. Then, click on "Edit Policy".
+4. Add the policy found in `policies/elasticsearch-policy.json` file in the cgp-dss-azul-indexer repository (Make sure to change the `Resource` value to the ARN of your elasticsearch box)
+
+### 5. Deploying Indexer Lambda
+
+1. In the directory of your chalice function, run `chalice deploy --no-autogen-policy`. Since we have created a policy in AWS we do not want chalice to automatically create a policy for us.
```

Comment:
Include same note regarding use of "--stage" option here as in section 3 step 4 above.